ts/nni_manager/rest_server/restHandler.ts

```diff
@@ -318,6 +327,67 @@ class NNIRestHandler {
         });
     }
 
+    private startTensorboardTask(router: Router): void {
+        router.post('/tensorboard', (req: Request, res: Response) => {
+            this.tensorboardManager.startTensorboardTask(req.body).then((taskDetail: TensorboardTaskInfo) => {
+                this.log.info(taskDetail);
+                res.send(Object.assign({}, taskDetail));
+            }).catch((err: Error) => {
+                this.handleError(err, res, false, 400);
+            });
+        });
+    }
+
+    private getTensorboardTask(router: Router): void {
+        router.get('/tensorboard/:id', (req: Request, res: Response) => {
+            this.tensorboardManager.getTensorboardTask(req.params.id).then((taskDetail: TensorboardTaskInfo) => {
+                res.send(Object.assign({}, taskDetail));
+            }).catch((err: Error) => {
+                this.handleError(err, res);
+            });
+        });
+    }
+
+    private updateTensorboardTask(router: Router): void {
+        router.put('/tensorboard/:id', (req: Request, res: Response) => {
+            this.tensorboardManager.updateTensorboardTask(req.params.id).then((taskDetail: TensorboardTaskInfo) => {
+                res.send(Object.assign({}, taskDetail));
+            }).catch((err: Error) => {
+                this.handleError(err, res);
+            });
+        });
+    }
+
+    private stopTensorboardTask(router: Router): void {
+        router.delete('/tensorboard/:id', (req: Request, res: Response) => {
+            this.tensorboardManager.stopTensorboardTask(req.params.id).then((taskDetail: TensorboardTaskInfo) => {
+                res.send(Object.assign({}, taskDetail));
+            }).catch((err: Error) => {
+                this.handleError(err, res);
+            });
+        });
+    }
+
+    private stopAllTensorboardTask(router: Router): void {
+        router.delete('/tensorboard-tasks', (req: Request, res: Response) => {
+            this.tensorboardManager.stopAllTensorboardTask().then(() => {
+                res.send();
+            }).catch((err: Error) => {
+                this.handleError(err, res);
+            });
+        });
+    }
```

Comment:
when users click `Clear all` button on webui, should we delete all tensorboard message at once? Looks like these stopped tensorboard are meaningless.

Reply:
del `STOPPED` tensorboard task info, but not `ERROR`.

Reply:
I have no other questions, thanks!